src/module-1/cards-list-v1/css/style.min.css

```diff
@@ -0,0 +1 @@
+@import url(https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap);*{font-family:Montserrat,sans-serif;box-sizing:border-box;margin:0;padding:0}.cards{padding-top:28px;display:flex;flex-wrap:wrap;justify-content:space-between;row-gap:16px}.cards .card{width:400px;border-radius:8px;background:#fff;padding-top:31px;box-shadow:0 2px 8px rgba(0,0,0,.135216)}.cards .card__wrapper{padding:0 23px;display:flex;flex-direction:column;align-items:center}.cards .card__img{max-width:60%}.cards .card__rate-price{padding-top:34px;width:100%;display:flex;flex-direction:row;justify-content:space-between}.cards .card__rate-price .rating{padding:5px;font-weight:400;font-size:18px;line-height:21px;color:#fff;background-color:#6f64f8;border-radius:5px}.cards .card__rate-price .rating span{font-size:14px;line-height:21px;padding:10px}.cards .card__rate-price .price{padding:5px;font-weight:400;font-size:18px;line-height:21px;color:#2c2c2c}.cards .card__title{padding-top:25px;font-weight:400;font-size:16px;line-height:19px;color:#2c2c2c;text-align:left;width:100%}.cards .card__descr{padding-top:3px;font-weight:300;font-size:12px;line-height:14px;color:#2c2c2c;text-align:left;width:100%}.cards .card__buttons{padding-top:22px;width:100%;display:flex;align-items:center}.cards .card__buttons .wishlist{display:flex;justify-content:center;align-items:center;padding:14px 0;background-color:#ededed;font-weight:600;font-size:14px;line-height:18px;color:#2c2c2c;text-transform:uppercase;width:50%;border:none;border-radius:0 0 0 8px}.cards .card__buttons .wishlist img{padding-right:10px}.cards .card__buttons .to-cart{display:flex;justify-content:center;align-items:center;padding:14px 0;background-color:#6f64f8;font-weight:600;font-size:14px;line-height:18px;color:#fff;text-transform:uppercase;width:50%;border:none;border-radius:0 0 8px 0}.cards .card__buttons .to-cart img{padding-right:10px}
```

Comment:
provide not .min.css just normal .css file on review its not production you don`t need to minify css